flask-server/avenv/lib/python3.12/site-packages/click/_compat.py

```
@@ -0,0 +1,622 @@
+class _AtomicFile:
+    def __init__(self, f: t.IO[t.Any], tmp_filename: str, real_filename: str) -> None:
+        self._f = f
+        self._tmp_filename = tmp_filename
+        self._real_filename = real_filename
+        self.closed = False
+
+    @property
+    def name(self) -> str:
+        return self._real_filename
+
+    def close(self, delete: bool = False) -> None:
```

Ruff (0.13.2)

Unused method argument: `delete`

(ARG002)